lib/web/ui/integration.go

```diff
@@ -234,6 +234,33 @@ type AWSOIDCDeployDatabaseServiceResponse struct {
 	ClusterDashboardURL string `json:"clusterDashboardUrl"`
 }
 
+// AWSOIDCEnrollEKSClustersRequest is a request to ListEKSClusters using the AWS OIDC Integration.
+type AWSOIDCEnrollEKSClustersRequest struct {
+	// Region is the AWS Region.
+	Region string `json:"region"`
+	// ClusterNames are names of the EKS clusters to enroll
+	ClusterNames []string `json:"clusterNames"`
```

Comment:
Should we also receive alias names for the clusters to import so they can be automatically named in Teleport with the user defined values?
If alias are not set, we import them with auto discovery names